### DISCUSSION TITLE
Release 4.1 & wifi on esp32_c3?

### DISCUSSION THREAD

**Discussion by tve** [Moddable-OpenSource/moddable#1214] [General]
I can't get either the wifican demo or the httpgetjson demo to work on an esp32 C3 (qtpyc3) using release 4.1. Both examples hang when the program starts
```
### Press RESET to restart device
#xsbug-log listening on port 5002. ^C to exit.
```
Both examples work fine using a moddable_two. I'm using ESP_IDF v4.4 which works great for me with moddable 3.9.4 on the esp32 c3.
Could some kind soul confirm whether the wifi/http examples work on a c3?

Update: looks like the idf-v5 branch works (with esp-idf v5) so probably some incompatibility between esp-idf v4.4 patch release versions...

**Comment by phoddie** [comment]
I just grabbed the ESP32-C3-DevKitM-1 board from the office shelf. It runs $MODDABLE/examples/network/http/httpget and $MODDABLE/examples/network/wifi/wifiscan just fine. I am using ESP-IDF v4.4.3 (which is the recommended version) - not sure if that will make a difference.

**Reply by tve** [reply]
Thanks a lot for taking the time to test! The idf_v5 branch is working for me now, so I'll stay on that. Did I read that it's about to be merged into public? I have a bunch of enhancements in my source dir that I'd like to create PRs from and if the merge is imminent I'd probably hold off 'til that happens.

**Reply by phoddie** [reply]
Yes, the IDF v5 merge is likely next week, to give it a little time to settle ahead of the October release.
